script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -592,47 +658,50 @@ Hey. Are you really okay with being with me? Aren't you afraid of slaying vampir
 [sha:528c8a4cd043d8163c7e1927a621a8837a8b6edc]{
 -- Page 337, Offset 20507.
 -- 唐突、というより今更な質問に、つい声をあげてしまった。
-I couldn't help but raise my voice in response to the sudden question. Or rather, to the fact that she asked me that now of all times.
+I couldn't help but raise my voice in response to the sudden question.
+ Or rather, in response to the fact that she's only now asking it.
 }
 [sha:cfe9b44858d3b80ec2e6f9dea7ec2463cda295f9]{
 -- Page 337, Offset 20508.
 -- アルクェイドと一緒にいて平気か、だって？
-She wants to know if I'm okay being with her?
+She wants to know if I mind being here with her?
 }
 [sha:ce894a8b6ea90eeb5b7e354f31ba61858258cbe7]{
 -- Page 337, Offset 20509.
 -- そんなの考えるまでもない。
-I don't even need to think to come up with an answer.
+I know the answer to that without even having to think about it.
 }
 [sha:03ba3908c4d80226189f59a9049c70ec76cb02c9]{
 -- Page 338, Offset 20510.
 -- 「別に。平気かどうかなんて、ただのえり好みだろ。
-"I guess. Regardless of whether it's okay or not, I just do it because I want to.
+"I don't mind. Being okay with it or not is splitting hairs, though.
 }
 [sha:6000c0a028067b34af7e3ebeeba9f1e6f07d739e]{
 -- Page 338, Offset 20511.
 -- 俺たちは利害が一致した協力関係なんだし。そりゃ吸血鬼は怖いけど、街の為なら我慢するさ」
-Our partnership is based on mutual interest after all. Of course I'm afraid of vampires, but I'll put up with it for the sake of this city."
+Our partnership is based on mutual interest after all.
+ Of course I'm afraid of vampires, but I'll put up with it for the sake of this city."
 }
 [sha:cf2a8e7623d70aa71c6dbfb29918be571166c372]{
 -- Page 339, Offset 20512.
 -- ふーん、と納得するアルクェイド。
-'Hmm', mutters Arcueid as if she was convinced.
+'Hmm', mutters Arcueid, seemingly convinced.
 }
 [sha:70b43be700babcf3a392c87d3a8cd479f2928606]{
 -- Page 339, Offset 20513.
 -- 会話はそこで途切れた。
-From there, the conversation comes to an end.
+With that, our conversation comes to a halt.
 }
 [sha:cf3c2f850d1ce6c24fac44039efc6ac5841dd27b]{
 -- Page 339, Offset 20514.
 -- 俺たちは無言で、すれ違う者のいない歩道を歩いていく。
-In complete silence, we walk down the sidewalk with no one passing by.
+In complete silence, we walk down deserted sidewalk.
 }
 [sha:46b861f2b25b7b5b18d5537d6c630f70b774b3ed]{
 -- Page 340, Offset 20515.
 -- 駅前から随分と離れた。この２時間、成果はほぼ無かったが不思議と徒労感はない。
-We've come a long way from the station. The past two hours have been almost completely lacking in results, but strangely enough, it didn't feel like a waste.
+We've come a long way from the station.
```

Comment:
This makes it sound almost like they've made great progress emotionally. I'd propose just "We're a long way from the station"